scroll-animations-1/Overview.bs

```diff
@@ -506,9 +484,11 @@ There are two types of scroll timeline offset: [=container-based offset=], and
 offset=] or to [=resolve an element-based offset=] depending on the offset type.
 It is possible for a [=scroll timeline offset=] to be resolved to null.
 
-The <dfn>effective start offset</dfn> is the [=effective scroll offset=] that is
-resolved from {{start}}. The <dfn>effective end offset</dfn> is
-the [=effective scroll offset=] that is resolved from {{end}}.
+The <dfn>effective start offset</dfn> is the value of first offset in
+[=effective scroll offsets=] array or null if the array is empty.
```

Comment:
What if the array has a single value in it? Shouldn't the effective offset be 'auto'?

Reply:
Nevermind, on rereading the algorithm for calculating effective scroll offsets, it looks like the one element offsets case becomes two effective scroll offsets.